packages/vm/tests/api/runCode.spec.ts

```diff
@@ -96,6 +98,71 @@ tape('VM.runCode: interpreter', (t) => {
   })
 })
 
+tape('VM: custom opcodes', (t) => {
```

Comment:
not sure if this new test suite belongs in runCode, i suggest we make a new file `tests/api/evm/customOpcodes.spec.ts`